src/Module.Server/ModuleData/characters.xml

```diff
@@ -1033,4 +1033,361 @@
     </Equipments>
   </NPCCharacter>
 
+  <NPCCharacter id="crpg_dtv_virgin"
+                  name="{=}Innocent Virgin">
+    <face>
+      <BodyProperties version="4"
+                      age="23"
+                      weight="0.3333"
+                      build="0"
+                      key="00026C0094FC300B78495C75178588756676A7368679B873ABBA698735DB468B014776130485B47400000000000000000000000000000000000000007E842104" />
+    </face>
+    <skills>
+      <skill id="Strength" value="15" />
+      <skill id="Agility" value="21" />
+      <skill id="IronFlesh" value="5" />
+      <skill id="PowerStrike" value="5" />
+      <skill id="PowerDraw" value="0" />
+      <skill id="PowerThrow" value="0" />
+      <skill id="Athletics" value="162" />
+      <skill id="Riding" value="0" />
+      <skill id="WeaponMaster" value="7" />
+      <skill id="MountedArchery" value="0" />
+      <skill id="Shield" value="0" />
+      <skill id="OneHanded" value="177" />
+      <skill id="TwoHanded" value="0" />
+      <skill id="Polearm" value="0" />
+      <skill id="Bow" value="0" />
+      <skill id="Crossbow" value="0" />
+      <skill id="Throwing" value="0" />
+    </skills>
+    <Equipments>
+      <EquipmentRoster>
+        <equipment slot="Head" id="Item.crpg_female_head_wrap" />

```

Comment:
I think all the items were recently renamed. If you took them from the website they are not up-to-date. Check `items.json` to find the latest update.

Reply:
You can use https://namidaka.fr for easy browsing with new ids . Right click open image in another tab to get id

Reply:
Thanks @namidaka, I was wondering why all the bots in warmup were naked!